drepl-usql/go.mod

```diff
@@ -1,42 +1,43 @@
 module github.com/astoff/drepl/drepl-usql
 
-go 1.22.1
+go 1.23
+
+toolchain go1.23.5
```

Comment:
Does this mean now that users who have an older Go will have trouble compiling the library? And why do you refer to a specific patch version (1.23.5)?

(Note that I'm not a Go programmer and especially the packaging stuff is a bit confusing to me. If you want to help make the experience smoother, including building some binaries in the CI so I can add the Lisp code to let the user download them without compiling their own, that's very welcome.)